java-client/src/main/java/org/scassandra/http/client/BatchStatement.java

```diff
@@ -0,0 +1,57 @@
+package org.scassandra.http.client;
+
+public final class BatchStatement {
```

Comment:
The spec calls them a query, I am guessing people will have too many classes called Query on their classpath so how about BatchQuery? This will need to represent PreparedStatements and Querys